### DISCUSSION TITLE
尝试进行Lora训练时，遇到加载本地 JSONL 文件时的 DatasetNotFoundError

### DISCUSSION THREAD

**Discussion by miaotouy** [zai-org/ChatGLM3#901] [Q&A]
### 试着在wsl中进行Lora训练，作为小白靠着LLM们的指点在尝试，到这个地方卡住了，有根据GPT的指点修改训练脚本和配置文件但不起作用。

要是提供的信息不够请告诉我补上

声明：使用AI生成大部分正文内容

**问题描述**
在尝试使用 `datasets` 库加载本地 `.jsonl` 文件进行微调时遇到错误。无论是使用相对路径还是绝对路径，`load_dataset` 函数都报告找不到数据集，好像它在尝试从 Hugging Face Hub 中加载数据集，而不是本地文件系统。

**错误信息**
```
DatasetNotFoundError: Dataset 'jsonl' doesn't exist on the Hub. If the repo is private or gated, make sure to log in with `huggingface-cli login`.
```

**复现步骤**
1. 确认数据文件 `merge_datasets.jsonl` 位于 `/mnt/e/rc20/ToolsRC20/ChatGLM3/ChatGLM3/finetune_demo/data` 目录中。
2. 在 WSL Ubuntu 22.04 LTS 环境中，通过 JupyterLab 终端执行以下命令：

```bash
CUDA_VISIBLE_DEVICES=0 python3 /mnt/e/rc20/ToolsRC20/ChatGLM3/ChatGLM3/finetune_demo/finetune_hf.py /mnt/e/rc20/ToolsRC20/ChatGLM3/ChatGLM3/finetune_demo/data /mnt/e/rc20/ToolsRC20/ChatGLM3/models/THUDM_chatglm3-6b /mnt/e/rc20/ToolsRC20/ChatGLM3/ChatGLM3/finetune_demo/configs/lora.yaml
```

**期望行为**
预期 `load_dataset` 函数能够正确加载本地 `.jsonl` 文件，而不是尝试从 Hugging Face Hub 加载数据集。

**请求帮助**
希望了解如何正确使用 `datasets` 库来加载本地 `.jsonl` 文件。是否存在需要调整的 `load_dataset` 函数调用方式或参数？

**环境信息**
- Windows 10 专业工作站版 22H2
- 使用 WSL Ubuntu 22.04.3 LTS 版本号 2204.3.63.0
- Python 版本: 3.10.12
- 在 JupyterLab 终端内运行命令
- 硬件信息: AMD Ryzen 9 5900X, NVIDIA GeForce RTX 4070 Ti SUPER, 64GB DDR4 RAM

**jsonl数据集每行内容的示例（有些行包含多轮）**
```
{"conversations": [{"role": "user", "content": "你还有同类吗？"}, {"role": "assistant", "content": "唔...Kafu我这种类型的灵魂现在似乎只有我一个呢。"}]}
```

**Comment by hnxtcyj123** [comment]
请问你解决了吗？我也遇到了同样的问题
